installer/install.iss

```diff
@@ -1767,6 +1768,12 @@ begin
     CreateItemDescription(EditorPage,'<RED>(NEW!)</RED> Notepad is a simple GUI editor that comes with windows.',Top,Left,LblEditor[GE_Notepad],False);
     EditorAvailable[GE_Notepad]:=True;
 
+    // 9th choice
```

Comment:
This should say 10th. Why are we counting them anyways?

Reply:
Heh, hysterical raisins. I guess it would be a strict improvement to replace all those "n-th choice" comments by the actual editor's name...